backend/spec/export_ead_spec.rb

```diff
@@ -1322,9 +1322,153 @@ def description_content(obj)
     it "will replace MSWord-style smart quotes with ASCII characters" do
       expect(serializer.remove_smart_quotes(note_with_smart_quotes)).to eq("This note has \"smart quotes\" and \'smart apostrophes\' from MSWord.")
     end
-  end
 
+    context 'when revision statement fields contain ampersand' do
+      let(:resource) do
+        create(:json_resource,
+          :publish => true,
+          :revision_statements => [
+            {
+              :date => 'date1&date1',
+              :description => '111111111&111111111',
+              :publish => true
+            },
+            {
+              :date => 'date2 & date2',
+              :description => '222222222 & 222222222',
+              :publish => true
+            }
+          ]
+        )
+      end
+
+      it "replaces ampersand with &amp; for revsion statement" do
+        document = get_xml("/repositories/#{$repo_id}/resource_descriptions/#{resource.id}.xml")
+        document_xml_string = document.to_xml
+
+        expect(document_xml_string).to_not include 'date1&date1'
+        expect(document_xml_string).to include 'date1&amp;date1'
+
+        expect(document_xml_string).to_not include '111111111&111111111'
+        expect(document_xml_string).to include '111111111&amp;111111111'
+
+        expect(document_xml_string).to_not include 'date2 & date2'
+        expect(document_xml_string).to include 'date2 &amp; date2'
+
+        expect(document_xml_string).to_not include '222222222 & 222222222'
+        expect(document_xml_string).to include '222222222 &amp; 222222222'
+      end
+    end
+
+    context 'when revision statement fields contain escaped ampersand in the form of &amp;' do
+      let(:resource) do
+        create(:json_resource,
+          :publish => true,
+          :revision_statements => [
+            {
+              :date => 'date1&amp;date1',
+              :description => '111111111&amp;111111111',
+              :publish => true
+            },
+            {
+              :date => 'date2 &amp; date2',
+              :description => '222222222 &amp; 222222222',
+              :publish => true
+            }
+          ]
+        )
+      end
+
+      it "replaces ampersand with &amp; for revsion statement" do
```

Comment:
Seems like this example should read "__does not replace__ ampersand with &amp; for revsion statement".